src/internal/Observable.ts

```diff
@@ -340,7 +340,8 @@ export class Observable<T> implements Subscribable<T> {
    * @method Symbol.observable
    * @return {Observable} this instance of the observable
    */
-  [Symbol_observable]() {
+  /** @ts-ignore mute errors for people without skipLibCheck on older versions of TS, remove this comment when we drop support for TS<4.3 */
```

Comment:
this has to be in a separate comment and right above the computed property for this to work 🤷‍♂️